### PR TITLE
feat(web): preview files in unzip file list

### PR DESCRIPTION
**Pull request by nourbalaha**
# Overview
In this PR I implemented the functionality to preview a file from unzip file list when this file is selected.

### CODE REVIEW

File: web/src/components/molecules/Asset/Asset/AssetBody/UnzipFileList.tsx [hunk 1]
```diff
@@ -38,7 +45,12 @@ const UnzipFileList: React.FC<Props> = ({ file, style }) => {
     setTreeData(getTreeData(file));
   }, [file, getTreeData]);
 
-  const onSelect: TreeProps["onSelect"] = (keys: Key[]) => {
+  const previewFile = (file: AssetFile) => {
+    setAssetUrl(assetBaseUrl + file.path);
+  };
```
Comment (KaWaite) [inline web/src/components/molecules/Asset/Asset/AssetBody/UnzipFileList.tsx:50]: useCallback

File: web/src/components/molecules/Asset/Asset/AssetBody/UnzipFileList.tsx [hunk 2]
```diff
@@ -38,7 +45,12 @@ const UnzipFileList: React.FC<Props> = ({ file, style }) => {
...
+
+  const onSelect: TreeProps<FileNode>["onSelect"] = (keys: Key[], { node: { file } }) => {
```
Comment (KaWaite) [inline web/src/components/molecules/Asset/Asset/AssetBody/UnzipFileList.tsx:52]: useCallback

**Review by KaWaite — APPROVED** [review]

File: web/src/components/molecules/Asset/Asset/AssetBody/UnzipFileList.tsx (hunk 1 above)
Comment (nourbalaha) [reply web/src/components/molecules/Asset/Asset/AssetBody/UnzipFileList.tsx:50]: Done!

File: web/src/components/molecules/Asset/Asset/AssetBody/UnzipFileList.tsx (hunk 2 above)
Comment (nourbalaha) [reply web/src/components/molecules/Asset/Asset/AssetBody/UnzipFileList.tsx:52]: Done!

**Pull request merged**